cmd/crypto/helpers.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * MinIO Cloud Storage, (C) 2018 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package crypto
+
+import (
+	"bytes"
+	"crypto/rand"
+	"io"
+	"io/ioutil"
+
+	"github.com/minio/sio"
+	"golang.org/x/crypto/argon2"
+)
+
+// IsEncryptedData - guess if the data is encrypted.
+func IsEncryptedData(data []byte) bool {
```

Comment:
```suggestion
func LikelyEncryptedData(data []byte) bool {
```

I think we should clearly indicate that this is a guess. With the values tested being `0x20, 0x01` there is a fairly big chance of a false positive. 1:65536 on random data.

An alternative would be to add a 4 byte header. That way we can easily detect it and also do versioning in the future if needed.

4 bytes is so unlikely that I can live with it.